src/Umbraco.Web.UI.Client/src/views/propertyeditors/nestedcontent/nestedcontent.controller.js

```diff
@@ -98,6 +98,8 @@
         vm.wideMode = Object.toBoolean(model.config.hideLabel);
         vm.hasContentTypes = model.config.contentTypes.length > 0;
 
+        var cultureChanged = eventsService.on('editors.content.cultureChanged', (_, args) => updateModel());
```

Comment:
Thanks @rickread! Can you explain what `(_, args) => updateModel()` does please? I am not super familiar with this notation. Is `_` using the underscore library or is it just a variable name? Apologies for my ignorance 😅

Reply:
Hi - my bad and no problem! I was basing this off an example, I've changed the callback signature to (name, args) to better reflect where eventsService is used elsewhere - it should return the name of the event being emitted, and any arguments to support the event being raised. In this case I'm only interested in the event being emitted so none of these are used, and when it occurs calling the updateModel method to force the nested content editor to load its model when the language variant changes.

Reply:
Using _ as a variable/parameter name is indicating that the value being passed/set will not be used. Much like how it's done in C# (although I think in C# it's actually part of the language, and the value is discarded)

Reply:
Thanks @rickread and @nathanwoulfe for the explanation, that makes more sense! So technically could this be `(_, _) => updateModel()` or is that not how it works in javascript? I learn new things every day still! 👍

Reply:
Not sure if the two same-named parameters would work - it's more a syntax thing than a language feature, think of it as indicating that the value in the parameter isn't anything we care about. 

If we didn't want any params, we send none - `() => someFunc()`. Using _ like the original example is useful when we want to ignore the first (or nth) param, but access others (n+1th, for example)

Javascript is weird.

Reply:
Haha! Fair enough, I've merged the current state, `() =>` would have been fine too, but this is good enough and does exactly the same! Thanks again for the teaching!